tests/testthat/test-release.R

```diff
@@ -199,3 +199,38 @@ test_that("get_release_data() works for new-style CRAN-RELEASE", {
   expect_equal(res$SHA, HEAD)
   expect_equal(path_file(res$file), "CRAN-SUBMISSION")
 })
+
+test_that("cran_version() is robust to unset CRAN mirror with unreleased pkg (#1857)", {
+  create_local_package()
```

Comment:
Instead of creating a local package, maybe we should just provide a nonsense package name to `cran_version()`, i.e. make this truly just a test of `cran_version()`.

Reply:
Hoping that no one releases a package with that name ;)